tests/functional/Glpi/Features/AssignableItemTest.php

```diff
@@ -73,43 +87,57 @@ public function testAddAndUpdateMultipleGroups(string $class): void
                 $class::getNameField() => __FUNCTION__ . ' 1',
                 'groups_id'            => [1, 2],
                 'groups_id_tech'       => [3],
+                'domains_id'            => getItemByTypeName(Domain::class, '_testDomain', true),
```

Comment:
if a `domain_id` is required for a specific class, then this case should be added in the `getMinimalCreationInput` method.